readthedocs/index.rst

```diff
@@ -93,3 +93,15 @@ Indices and tables
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+
+MSP file to a Spectrum object
+=============================
+
+Bellow is an example on how to parse a .msp file to a Spectrum object.
+
+import os
+import pytest
+from matchms.importing import load_from_msp
+
+spectrums = load_from_msp("MoNA-export-GC-MS.msp")
```

Comment:
Adding a code example is nice!
For readthedocs we should place it in the load_from_msp docstring though, then it will automatically be added (also easier to maintain later).
You can just add an example code block like the following:

``` python
""" docstring...

.. code-block:: python

    from matchms.importing import load_from_msp
    spectrums = load_from_msp("MoNA-export-GC-MS.msp")

"""
```

Reply:
Oh yes! I read about that but forgot completely, will do it next time.